src/lib/api/http.ts

```diff
@@ -52,6 +52,7 @@ export default async ( path: string, options: FetchOptions = {} ): Promise< Resp
 			'Content-Type': 'application/json',
 			...( options.headers ?? {} ),
 		},
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
Needed to quickly fix this lint error as it wasn't really related to my commit. Might be my overzealous git hook for catching this though.